activity_log.php

```diff
@@ -30,9 +30,6 @@
 $sys = ( $is_admin && getGetValue ( 'system' ) != '' );
 
 print_header();
-
-ob_start();
```

Comment:
"This branch has conflicts"
Damn. I've managed to corrupt the master on my fork.
The "fixes" I've tried just make it worse.
Is there a way to "re-fork" or re sync my master with yours?
Without creating a new account?